app/helpers/proxmox_vm_cloudinit_helper.rb

```diff
@@ -48,7 +48,7 @@ def create_cloudinit_iso(vm_name, configs, ssh)
     wd = create_temp_directory(ssh)
 
     configs.each do |config|
-      config_file = ssh.run(%(echo '#{config[1]}' >> "#{wd}/#{config[0]}"))
+      config_file = ssh.run("cat <<'EOF' > '#{wd}/#{config[0]}'\n#{config[1]}\nEOF")
```

Comment:
the old implementation would ADD the config line. The new one, would not append but replace. what do we need?